servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/HttpProtocolConfigs.java

```diff
@@ -52,6 +54,11 @@ public static H1ProtocolConfigBuilder h1() {
     /**
      * Returns {@link H2ProtocolConfig} with the default configuration for
      * <a href="https://tools.ietf.org/html/rfc7540">HTTP/2</a>.
+     * <p>
+     * Note this doesn't necessarily provide {@link H2ProtocolConfig#initialSettings()} that corresponds to
```

Comment:
worth calling out for visibility...  default values were set based upon experience and this carve out is meant to highlight that folks should be explicit if they depend upon specific values (defaults are subject to change).